pennylane/templates/broadcast.py

```diff
@@ -65,7 +65,7 @@ def wires_all_to_all(wires):
 
 # define wire sequences for patterns
 PATTERN_TO_WIRES = {
-    "single": lambda wires: [wires[i] for i in range(len(wires))],
+    "single": lambda wires: [wires.subset([i]) for i in range(len(wires))],
```

Comment:
Snuck this in here: the change of indexing makes this pattern behave differently from multi-wire ones. Should be safe and helps to fix #1030.

Reply:
👍

Reply:
🔥